-standup-template.md

```diff
@@ -2,10 +2,11 @@
 
 <sup>[from template][standup-template] | [:notebook: Meetings][meetings] | [:date: Calendar][calendar] | [:white_check_mark: Tasks][tasks] | [:cat: Code][gh] | [:open_file_folder: Drive][gdrive]</sup>
 
-Time:      ...  
-Location:  https://meet.jit.si/hyphacoop  
-Attending: ...  
-Notetaker: :raising_hand: el, yj, gi, pc, dc, uv, bl  
+Time:       ...  
+Location:   https://meet.jit.si/hyphacoop  
+Attending:  ...  
+Cultivator: ...
```

Comment:
Not sure if this is really important information to keep record but ok adding.

Reply:
Because I was going back to see who'd taken turns and when, and it wasn't recorded anywhere clear. (To know if I should feel its time to volunteer again or what)